### PR TITLE
Update-to-Milestone-2-MyPortfolio-Project

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
Update after revision of layout based on Figma web page, as part of the Day 4 - Week 1 - Activity at Microverse.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Hamzaoutdoors) [inline README.md:57]: - The `MIT` link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```
Comment (Hamzaoutdoors) [inline index.html]: - [x] Please kindly fix this image `src`, as you can notice in the screenshot below the image is broken, the `src` is src="./images/background.jpg" with one point to track the path for your image

Capture

File: index.html [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+
+      <div>
+        <button type="button" class="my-project-button">See Project</button>
+      </div>
+    </div>    
+
+    <div class="my-projects">
+      <div class="my-project-image-container">
+        <img
+          src="../images/background.jpg"
```
Comment (Hamzaoutdoors) [inline index.html]: The same bug requested in the comment below, Please, fix the image problem by changing the `src` to `src="./images/background.jpg"`

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @fernando-silvabr66, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
**N.A**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (fernando-silvabr66) [reply README.md:57]: Thanks for the heads up.

File: index.html (hunk 3 above)
Comment (fernando-silvabr66) [reply index.html]: Fixed also... thanks.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟨 Status: Changes Required 🟥 

### Hi @fernando-silvabr66 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As pointed out by the previous reviewer, the project images do not show up in the live demo, please refer to the previous reviewers comments, fix the problem and let your portfolio shine 🌞 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fernando-silvabr66** [comment]
Hi, I have changed the src configuration for the images  (it's appearing in
my Live server).
Do you have any additional suggestions (please)?

Regards.

Em sáb., 8 de jan. de 2022 23:09, Nakitto Catherine <
***@***.***> escreveu:

**Comment by fernando-silvabr66** [comment]
Sorry fot the mistyping... word corrector...

Em sáb., 8 de jan. de 2022 23:15, Fernando Silva <
***@***.***> escreveu:

**Comment by fernando-silvabr66** [comment]
Due to time restrictions, I used an alternative solution in order to have it submitted in time... thanks.

**Review by Cathella — APPROVED** [review]
## 🟩 Status: Project Approved 🎉 

### Hi @fernando-silvabr66 👋🏻, 

Good job making all the requested changes 💪🏻.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional Changes ♻️ 

- [ ] [OPTIONAL] Please try to center the rest of the images too so that you have a consistent design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fernando-silvabr66** [comment]
Finalized... cheers!

**Pull request merged**